examples/nested/values.yaml

```diff
@@ -0,0 +1,24 @@
+spire-server:
+  enabled: true
+  upstreamAuthority:
+    spire:
+      enabled: true
```

Comment:
This value is duplicated in line 22, leading to the ability to get it wrong in one or both places.  This is the example that proves the design of the YAML values is wrong.

Reply:
Whether to have an upstream spire authority config can be separate from deploying it with this chart.